README.md

```diff
@@ -1,29 +1,71 @@
-# CNTK Object Recognition Video Tagging Tool
+# CNTK Object Detection Video Tagging Tool
 
-This tool provides end to end support for generating datasets for and validating Object Recognition Models with CNTK.
+This tool provides end to end support for generating datasets for and validating Object Detection Models with CNTK.
 
 It supports the following scenarios:
 
 - Computer Assisted Tagging of Objects In Video Using Custom implementation of Camshift Algorithm
-- Export Tags to CNTK format for training a CNTK object recognition model.
+- Exporting Tags to CNTK format for training a CNTK object recognition model.
 - Running and validating a trained CNTK object recognition model on new videos to generate stronger models. (Windows only for now, GPU recommended)
 
-## To Use:
- 1. Download the release binary.
- 2. Extract and run the app
- 3. Load a video.
- 4. Configure the tagging job specifying the following preferences
+## Table of Contents
+ - [Prerequisites](#prerequisites)
+ - [Tagging Job](#taggingjob)
+ - [Review/Itterate CNTK Object Detction Model](#review)
+ - [Upcoming Features](#upcoming)
+ 
+<a name="prerequisites"></a>
+## Prerequisites
+ 1. Install [CNTK and Dependencies](https://github.com/Microsoft/CNTK/wiki/Object-Detection-using-Fast-R-CNN#setup)
```

Comment:
It's worth mentioning here that this is only needed in case you want to review the mode.